html/includes/api_functions.inc.php

```diff
@@ -112,19 +200,13 @@ function get_port_stats_by_port_hostname()
         }
     }
 
-    $output    = array(
-        'status' => 'ok',
-        'port'   => $port,
-    );
-    $app->response->headers->set('Content-Type', 'application/json');
-    echo _json_encode($output);
+    api_success($port, 'port');
 }
 
 
 function get_graph_generic_by_hostname()
 {
     // This will return a graph type given a device id.
-    global $config;
```

Comment:
This is still needed for the pages that return graphs as we use `$config` within the included files.